.chloggen/1440.yaml

```diff
@@ -0,0 +1,4 @@
+change_type: enhancement
+component: feature_flag
+note: Define `feature_flag.evaluation` event.
```

Comment:
Rename `feature_flag` event to `feature_flag.evaluation` event, define new feature flag attributes and provide body definition. Remove `feature_flag` span event definition in favor of log-based event.